Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py

```diff
@@ -356,6 +356,9 @@ def parse_args(args):
         optparse.make_option(
             "--use-async-uikit-interactions", action="store_true", default=False,
             help=("Opt into async UIKit interactions (iOS-family WebKit2 ports only)")),
+        optparse.make_option(
+            "--no-use-async-uikit-interactions", action="store_true", default=False,
+            help=("Opt out of async UIKit interactions (iOS-family WebKit2 ports only)")),
```

Comment:
Why do we need both options?

Reply:
I'm adding both options so that on a simulator with `UIKit/async_text_input=on`, it's easy to A/B test for correctness by doing:

```
run-webkit-tests … editing/selection --use-async-uikit-interactions
run-webkit-tests … editing/selection --no-use-async-uikit-interactions
```

(Actually, now that I think about it more — there's no need for `--use-async-uikit-interactions` anymore, since we can just omit the argument. I'll make that adjustment, good catch!)